.github/workflows/operate-ci-test-reusable.yml

```diff
@@ -73,6 +74,11 @@ jobs:
           nexusUsername: ${{ steps.secrets.outputs.NEXUS_USR }}
           nexusPassword: ${{ steps.secrets.outputs.NEXUS_PSW }}
 
+      # Build: all Maven artifacts
+      - name: Build Maven
+        run: |
+          mvn clean install -B -T1C -P -docker,skipFrontendBuild -DskipTests -DskipChecks -DaltStagingDirectory=${{ github.workspace }}/staging-${{ env.BRANCH_NAME }} -DskipRemoteStaging=true -Dmaven.deploy.skip=true
+
```

Comment:
❓ ah this was necessary to build the dist?

Reply:
Some integration tests require the class `StandaloneOperate`, which is now in the `dist` module (previously called `Application` in the `operate-webapp` module). So far, the Operate CI only built and tested the Operate modules. 

With that additional step, we would ensure that the entire project is built and everything is present, i.e., the `dist` module is built so that the integration tests can be executed properly.